apps/predbat/predbat.py

```diff
@@ -128,6 +128,34 @@
     {'name' : 'holiday_days_left',             'friendly_name' : 'Holiday days left',              'type' : 'input_number', 'min' : 0,   'max' : 28,    'step' : 1,    'unit' : 'days', 'icon' : 'mdi:clock-end'},
 ]
 
+INVERTER_DEF={ 
+    "GE": {
+        'has_rest_api': True,
+        'output_charge_control': "current",
```

Comment:
These don't like right for GE, in that it has a change and discharge time enable and a target soc?